image/package.json

```diff
@@ -27,8 +27,7 @@
         "lint": "tslint -c ../tslint.json -p . -t verbose",
         "build-demos": "cd demos && ./build-demos.sh",
         "run-demos": "yarn build-demos && http-server demos/",
-        "publish-demos": "yarn build-demos && mkdir -p ../docs/image/demos && cp demos/*.{js,html,css,jpg} ../docs/image/demos",
-        "doc": "typedoc --sourcefile-url-prefix 'https://github.com/tensorflow/magenta-js/tree/master/image/src/arbitrary_stylization/' --out ../docs/image src --mode file --excludePrivate --exclude '**/*+(index|test).ts' --excludeExternals && yarn publish-demos"
+        "docs": "sh ../scripts/generate-docs.sh 'image' 'https://github.com/tensorflow/magenta-js/tree/master/image/src/arbitrary_stylization/' 'file'"
```

Comment:
i'm not 100% convinced why this file couldn't also be "modules", btw. have you tried? then you wouldn't have to pass this in